pkgs/tools/audio/bpm-tools/default.nix

```diff
@@ -12,15 +22,18 @@ stdenv.mkDerivation rec {
     sha256 = "151vfbs8h3cibs7kbdps5pqrsxhpjv16y2iyfqbxzsclylgfivrp";
   };
 
-  patchPhase = ''
-    patchShebangs bpm-tag
-    patchShebangs bpm-graph
-  '';
+  nativeBuildInputs = [ makeWrapper ];
 
   installFlags = [
     "PREFIX=${placeholder "out"}"
   ];
 
+  postFixup = ''
+    wrapProgram $out/bin/bpm --prefix PATH : "${path}"
```

Comment:
Technically, I think this executable doesn't need wrapping as it's only usable when piped to, as in the example `./result/bin/bpm -h` shows:

```
sox file.mp3 -t raw -r 44100 -e float -c 1 - | ./bpm
```

Reply:
Correct. Resolved it.